Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: clean install analysis test test-install develop docs docs-install
 
 venv:
-	virtualenv venv
+	virtualenv --python=python venv
```

Comment:
Adding `--python=python` uses the standard alias Travis-CI sets up in each environment, which seemed to be the simplest way to use the intended test version. I can look around for another way if you think there is one.